src/org/openlcb/can/MessageBuilder.java

```diff
@@ -239,6 +239,11 @@ List<Message> processFormat1(CanFrame f) {
             case InitializationComplete:
                 retlist.add(new InitializationCompleteMessage(source));
                 return retlist;
+
+            case InitializationCompleteSimple:
```

Comment:
could you add unit tests into https://github.com/openlcb/OpenLCB_Java/blob/master/test/org/openlcb/can/MessageBuilderTest.java

We need two tests per message type: one rendering messages into can frames and one p[arsing a can frame into the message.